src/canvas.py

```diff
@@ -52,7 +52,7 @@ def __init__(self, application):
                      self.top_right_axis]:
             axis.get_xaxis().set_visible(False)
             axis.get_yaxis().set_visible(False)
-        DummyToolbar(self)
+        self.dummy_toolbar = DummyToolbar(self)
```

Comment:
```suggestion
        DummyToolbar(self)
```
This might be intransparent, but upon init, the Toolbar automatically sets itself as the toolbar attribute of the canvas provided, making a second assignment redundant

Reply:
Thanks, I was slightly unsure if the attribute would persist, but it's good without the explicit assignment then.